README.md

```diff
@@ -1,115 +1,38 @@
-# plugin-&lt;REPLACE ME&gt;
+# Salesforce DX
```

Comment:
Make sure you do NOT set up circleci for this repo yet. The current state of the jobs will not support releasing salesforcedx.